Os.py

```diff
@@ -0,0 +1,10 @@
+#python defined packages
+import sys
+import argparse
```

Comment:
Why argparse go with JSON it will be fast

Reply:
Changes Done